eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/FastList.java

```diff
@@ -262,18 +263,33 @@ public int getBatchCount(int batchSize)
         return this.toArray((E[]) new Object[sourceToIndex - sourceFromIndex + 1], sourceFromIndex, sourceToIndex, 0);
     }
 
+    /**
+     * Overrides default method from List.
+     *
+     * @since 10.0
+     */
+    @Override
+    public void sort(Comparator<? super T> comparator)
+    {
+        if (this.size > 1)
+        {
+            Arrays.sort(this.items, 0, this.size, comparator);
+        }
+    }
```

Comment:
It's not clear to me where this was moved from. The superclass doesn't know it can delegate to the backing array, right? What was the old behavior?